src/main/java/mate/jdbc/model/Manufacturer.java

```diff
@@ -0,0 +1,45 @@
+package mate.jdbc.model;
+
+public class Manufacturer {
```

Comment:
this class constrains constructors and setters looks messy and redundant.
Better to use one approach for initializing java object constructor init or setters methods.
So please choose one approach and cleaned up this class and code as well